code-studio/src/js/components/CellEditor.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * @overview React component to allow for easy editing and creation of
+ * Cells
+ * @see @cdo/apps/maze/cell
+ */
+/* global React */
+
+var Cell = require('@cdo/apps/src/maze/cell');
+
+module.exports = React.createClass({
```

Comment:
I like to name the class, otherwise it shows up in various tools as module.exports, i.e.
`var CellEditor = module.exports = React.createClass({`
or
`var CellEditor = React.createClass({
 ...
 module.exports = CellEditor
`